mysql-test/suite/versioning/r/vtmd.result

```diff
@@ -358,6 +358,82 @@ set versioning_alter_history= survive;
 create or replace table t (x int) with system versioning;
 select * from t for system_time all;
 x	sys_trx_start	sys_trx_end
+drop database test;
+create database test;
+use test;
+set versioning_alter_history = survive;
+create or replace table t (fire int) with system versioning;
+insert into t values (1);
+set @ts1= now(6);
+alter table t add column ice int;
+update t set fire= 2;
+set @ts2= now(6);
+alter table t drop column ice;
+update t set fire= 3;
+create or replace procedure select_a_from_range()
+begin
+select fire from t for system_time from '0-0-0' to now(6);
+select fire from t for system_time between '0-0-0' and now(6);
+select fire from t for system_time as of @ts2;
+end~~
+set versioning_identifier_mode= current;
+call select_a_from_range();
+fire
+1
+fire
+1
+fire
+2
+set versioning_identifier_mode= historical_early;
+call select_a_from_range();
+fire
+1
+fire
+1
+fire
+2
+set versioning_identifier_mode= historical;
+call select_a_from_range();
+fire
+3
+2
+fire
+3
+2
+fire
+2
+drop database test;
+create database test;
+use test;
+set versioning_alter_history= survive;
+set versioning_identifier_mode= current;
+create or replace table t (fire int) with system versioning;
+insert into t values (1);
+set @ts= now(6);
+alter table t add column ice int first, drop column fire;
+select fire from t for system_time as of @ts;
+fire
+1
+select ice from t for system_time as of @ts;
+fire
+1
```

Comment:
That's strange: same column is accessed by 2 different names. Expected error for wrong name.